### PR TITLE
[Added] Manual dismiss

### PR DESCRIPTION
**Pull request by carmelogallo**

### CODE REVIEW

File: Sources/FusumaViewController.swift [hunk 1]
```diff
@@ -340,8 +342,8 @@ public struct ImageMetadata {
         
         self.delegate?.fusumaWillClosed()
         
-        self.dismiss(animated: true) {
-        
+        self.doDismiss {
```
Comment (ichiki1023) [inline Sources/FusumaViewController.swift]: ```self.autoDismiss```
Does that fit this?

**Comment by carmelogallo** [comment]
Hi @ytakzk, could you merge this? Is really important for some of us. Thanks!

**Comment by ichiki1023** [comment]
@cikpis  ```autoDismiss``` is global variable.
How about using ```fusumaAutoDismiss``` instead of ```autoDismiss```.

**Comment by carmelogallo** [comment]
@ichiki1023 I don't get it. `fusumaAutoDismiss` is the one I created and it's global like `fusumaCropImage` that was there.

If you read the code I edited is really simple and functional. What did you mean?

**Comment by ichiki1023** [comment]
@cikpis I just want to know the reason for using autoDismiss instead of fusumaAutoDismiss.

**Comment by carmelogallo** [comment]
Sure, I understood another thing before and maybe now I got your question. 

Basically, I’m not presenting the Fusuma View Controller but I’m adding the view on my VC because I need to do other operations that are not part of Fusuma. 

So, managing the manual dismiss for me is better.

**Comment by carmelogallo** [comment]
If you are talking about variables, fusumaAutoDismiss was added from me in my PR.

**Comment by ichiki1023** [comment]
@cikpis   I see! Thanks!

**Comment by carmelogallo** [comment]
Nice!

**Pull request merged**